### ISSUE TITLE
Color Contrast Issue in Links

### ISSUE THREAD

**Issue by mgifford** [LiaScript/LiaScript#240]
**Title: WCAG 1.4.3: Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (#focusedToc)**
Tags: Accessibility, WCAG 1.4.3, color-contrast

Issue: Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (color-contrast - https://accessibilityinsights.io/info-examples/web/color-contrast)

Target application: Learner Instructions · Digital Acquisition Professional Program Case Study Challenge - https://liascript.github.io/course/?https://raw.githubusercontent.com/usds/ditap-curriculum-update/main/3_Curriculum/3A_DITAP-Core-Curriculum/1_Vendor-Materials/Case%20Study%3A%20Threaded%20Case%20Scenario/Core%20Case%20Study%3A%20Navigating%20Stakeholder%20and%20Decision-Making%20Challenges.md#8

Element path: #focusedToc

Snippet: <a tabindex="0" aria-hidden="false" class="lia-active lia-toc__link lia-toc__link--is-lvl-3" href="#8" id="focusedToc">Learner Instructions</a>

How to fix: 
Fix any of the following:
  Element has insufficient color contrast of 3.71 (foreground color: #399193, background color: #ffffff, font size: 11.3pt (15px), font weight: normal). Expected contrast ratio of 4.5:1

Environment: Microsoft Edge version 139.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.46.0 (axe-core 4.10.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

**Title: WCAG 1.4.3: Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (.notranslate[translate="no"])**
Tags: Accessibility, WCAG 1.4.3, color-contrast

Issue: Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (color-contrast - https://accessibilityinsights.io/info-examples/web/color-contrast)

Target application: Learner Instructions · Digital Acquisition Professional Program Case Study Challenge - https://liascript.github.io/course/?https://raw.githubusercontent.com/usds/ditap-curriculum-update/main/3_Curriculum/3A_DITAP-Core-Curriculum/1_Vendor-Materials/Case%20Study%3A%20Threaded%20Case%20Scenario/Core%20Case%20Study%3A%20Navigating%20Stakeholder%20and%20Decision-Making%20Challenges.md#8

Element path: button[aria-label="Translations"] > .notranslate[translate="no"]

Snippet: <span class="notranslate" translate="no">EN</span>

How to fix: 
Fix any of the following:
  Element has insufficient color contrast of 3.71 (foreground color: #399193, background color: #ffffff, font size: 15.0pt (20px), font weight: normal). Expected contrast ratio of 4.5:1

Environment: Microsoft Edge version 139.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.46.0 (axe-core 4.10.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

**Comment by andre-dietrich** [comment]
Can you try this out with dark and light background? The colors and the CSS were set by a company, I would like to change this to a better color, when it does not make the dark-mode worse?

Does the problem persist for the other color-schemes?

If you need it urgently, we could also support you with a custom color scheme. You can either use `@style` to overwrite styles or use it to overwrite some of the CSS variables (all variables start with `--`), you can link a custom style `link: url` within the header or use this fallback-macro custom, which will add the new variable definitions and add an additional design-button.

[Try the example in the LiveEditor](https://liascript.github.io/LiveEditor/?/show/code/H4sIAAAAAAAAA32PzwqDMAzG732KjF0NTMGLjiHsuttewGo7LVYjNWV/nn5VUMYGgySHfL8kX447RCGK2k9MvUCsyZLD1jStDckZJBEkaRrBId/UStZd48gPKoM4CcBaPhBySrsMDtES+e9mVNJ1f4FeDz4ciONvhPWD18ngbRYaS5W0eKOBcTIvHeac7mdla2HvLZvRmtlXsFroQYkAnITYw3n5P1BwMfJaOzMy3A23UBYTP60uhWBS9AZVE69SLgEAAA==)

``` markdown
<!--
@custom
--color-highlight: 2, 255, 0;
--color-background: 122, 122, 122;
--color-border: 0, 0, 0;
--color-highlight-dark: 0, 0, 0;
--color-highlight-menu: 111, 0, 0;
--color-text: 0, 0, 255;
--global-font-size: 1rem;
--font-size-multiplier: 2;
@end
-->

# Customize LiaScript with `@style`

todo
```

<img width="2500" height="1290" alt="Image" src="https://github.com/user-attachments/assets/bdea9ee8-cb12-4c77-bc42-7bb7d3d94438" />

**Comment by andre-dietrich** [comment]
As mentioned earlier, in dark-mode we will still have to update all CSS requirements, but in light-mode we should have fixed all color contrast issues ...
